cursor.h

```diff
@@ -0,0 +1,27 @@
+// Copyright 2021 Google LLC
+// Copyright 2021 @filterpaper
+// Copyright 2023 Pablo Martinez (@elpekenin) <elpekenin@elpekenin.dev>
+// Copyright 2024 Guillaume Stordeur <guillaume.stordeur@gmail.com>
+// Copyright 2024 Matt Skalecki <ikcelaks@gmail.com>
+// Copyright 2024 QKekos <q.kekos.q@gmail.com>
+// SPDX-License-Identifier: Apache-2.0
+// Original source/inspiration: https://getreuer.info/posts/keyboards/autocorrection
+#pragma once
+
+#include "keybuffer.h"
```

Comment:
could limit the use of header includes by including these in cursor.c before including cursor.h

Reply:
I followed this advice after your explanation in Discord